scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -81,6 +81,9 @@ abstract class H2OMOJOModel
     new NullableStringParam(this, "modelCategory", "H2O's model category")
   protected final val scoringHistory: NullableDataFrameParam =
     new NullableDataFrameParam(this, "scoringHistory", "Scoring history acquired during the model training.")
+  protected var crossValidationModelsScoringHistory =
+    new NullableDataFrameArrayParam(this, "crossValidationModelsScoringHistory", "Cross validation scoring history.")
```

Comment:
Would it possible to update description to have the same meaning as the parameter name?

Reply:
Documentation string fixed.